src/ahbicht/expressions/expression_resolver.py

```diff
@@ -0,0 +1,46 @@
+"""
+This module makes it possible to parse expressions with all its subexpressions
+for example ahb_expressions with condition_expressions or condition_expressions with packages.
+"""
+
+from lark import Transformer, Tree
+
+from ahbicht.expressions.ahb_expression_parser import parse_ahb_expression_to_single_requirement_indicator_expressions
+from ahbicht.expressions.condition_expression_parser import parse_condition_expression_to_tree
+
+
+def expression_resolver(expression: str, resolve_packages: bool = False) -> Tree:
```

Comment:
nachdem mir jetzt unten im test klar geworden ist, was das ding tut würde ich für die konsistenz vorschlagen, dass wir das wort "parse" auch in den namen packen, weil die anderen "high level" functions fangen auch alle mit `parse_...` an. Zum beispiel:
```
parse_ahb_or_condition_expression
```
?

Reply:
89c67888f45169382775bec71b9174dc5c268ca3